src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -120,6 +126,8 @@ internal static ErrorCode GetErrorCode(this LanguageVersion version)
                     return ErrorCode.ERR_FeatureNotAvailableInVersion7;
                 case LanguageVersion.CSharp7_1:
                     return ErrorCode.ERR_FeatureNotAvailableInVersion7_1;
+                case LanguageVersion.CSharp7_2:
+                    return ErrorCode.ERR_FeatureNotAvailableInVersion7_2;
```

Comment:
>ERR_FeatureNotAvailableInVersion7_2 [](start = 37, length = 35)

What is the benefit of having an error code per version? It looks like we could easily use one parameterized error message. In fact, some features in some situations we do this (use parameterized error message) when these errors cannot be used.